HPXMLtoOpenStudio/tests/test_water_heater.rb

```diff
@@ -1048,6 +1050,40 @@ def test_tank_heat_pump_capacities
     assert_in_epsilon(cap, coil.ratedHeatingCapacity, 0.01)
   end
 
+  def test_tank_heat_pump_containment_volume_adjustment
+    # volumes and their corresponding expected cops are based on RESNET spreadsheet: ConstrainedHPWH_04xlsx.xlsx
+    containment_volumes = [3000, 1500, 960, 707, 453, 200, 83.5]
+    expected_cop = [3.720, 3.720, 3.647, 3.518, 3.191, 2.367, 1.642]
```

Comment:
I'm using the RESNET spreadsheet to calculate the adjusted COPs while changed the original COP to be 3.73135, which is our test file configurations. All the other changes in this file are made so that we can configure test files on the fly and verify EnergyPlus inputs.

Reply:
For the link to the standard itself, just link to the final draft available at time of implementation (e.g., [this](https://www.resnet.us/about/standards/hers/draft-pds-03-hers-addendum-77-integrated-heat-pump-water-heaters-ihpwh/)) and we can update it down the road when it's officially published.